lib/Runtime/Library/ArrayBuffer.h

```diff
@@ -159,7 +159,8 @@ namespace Js
         virtual BOOL GetDiagTypeString(StringBuilder<ArenaAllocator>* stringBuilder, ScriptContext* requestContext) override;
         virtual BOOL GetDiagValueString(StringBuilder<ArenaAllocator>* stringBuilder, ScriptContext* requestContext) override;
 
-        virtual ArrayBufferDetachedStateBase* DetachAndGetState();
+        void Detach();
```

Comment:
or protected.

---
In reply to: [130999054](https://github.com/Microsoft/ChakraCore/pull/3468#discussion_r130999054) [](ancestors = 130999054)